### PR TITLE
Update multisigs info

### PR DESCRIPTION
**Pull request by kate-aleksseeva**

### CODE REVIEW

File: docs/multisigs/committees.md [hunk 1]
```diff
@@ -10,11 +10,15 @@ The LEGO Committee utilizes this multisig to make grants: sending dedicated gran
 
 **Address:** [`0x12a43b049A7D330cB8aEAB5113032D18AE9a9030`](https://app.safe.global/settings/setup?safe=eth:0x12a43b049A7D330cB8aEAB5113032D18AE9a9030)
 
-**Forum topic:** [Project: Lido Ecosystem Grants Organization](https://research.lido.fi/t/project-lido-ecosystem-grants-organization/406), [Proposal to continue LEGO fro Q1 2022](https://research.lido.fi/t/lego-a-proposal-to-continue-lego-for-q1-2022/1568/)
+**Forum topics:**\
+[Project: Lido Ecosystem Grants Organization](https://research.lido.fi/t/project-lido-ecosystem-grants-organization/406)\
+[Proposal to continue LEGO fro Q1 2022](https://research.lido.fi/t/lego-a-proposal-to-continue-lego-for-q1-2022/1568/)
```
Comment (AlexLido) [inline docs/multisigs/committees.md:13]: for Q1

File: docs/multisigs/committees.md [hunk 2]
```diff
@@ -124,7 +132,7 @@ MEVBoostRelayAllowedList [`0xF95f069F9AD107938F6ba802a3da87892298610E`](https://
 
 **Snapshot:** [Proposal to form TRP Committee](https://snapshot.org/#/lido-snapshot.eth/proposal/0xc00b48275e268f26b6cebf82322f281a44acaf679f6381dd612e278174671daf)
```
Comment (AlexLido) [inline docs/multisigs/committees.md:127]: Missing line break?

File: docs/multisigs/committees.md [hunk 3]
```diff
@@ -124,7 +132,7 @@ MEVBoostRelayAllowedList [`0xF95f069F9AD107938F6ba802a3da87892298610E`](https://
...
 
-**On-chain:** [Omnibus Vote #152 to set up LDO Easy Track](https://vote.lido.fi/vote/152)
+**Aragon:** [Omnibus Vote #152 to set up LDO Easy Track](https://vote.lido.fi/vote/152)
```
Comment (AlexLido) [inline docs/multisigs/committees.md]: Missing line break?

File: docs/multisigs/committees.md [hunk 4]
```diff
@@ -125,7 +125,7 @@ The full list of related contracts is available [here](https://docs.lido.fi/depl
 
 **Snapshot:** [Proposal to form TRP Committee](https://snapshot.org/#/lido-snapshot.eth/proposal/0xc00b48275e268f26b6cebf82322f281a44acaf679f6381dd612e278174671daf)
 
-**Aragon:** [Omnibus Vote #152 to set up LDO Easy Track](https://vote.lido.fi/vote/152)
+**Aragon:** [Omnibus vote #152 to set up LDO Easy Track](https://vote.lido.fi/vote/152)
```
Comment (AlexLido) [inline docs/multisigs/committees.md:129]: Missing line break?

File: docs/multisigs/committees.md [hunk 5]
```diff
@@ -361,7 +361,7 @@ This Multisig is allowed to [hold stETH](https://research.lido.fi/t/rewards-janu
 
 **Addresses:**\
 `0xC18F11735C6a1941431cCC5BcF13AF0a052A5022` - [Ethereum](https://app.safe.global/home?safe=eth:0xC18F11735C6a1941431cCC5BcF13AF0a052A5022), [Arbitrum](https://app.safe.global/home?safe=arb1:0xC18F11735C6a1941431cCC5BcF13AF0a052A5022), [Base](https://app.safe.global/home?safe=base:0x4f793e5d1d71dbbcEE34E39A5aD3c6bA5b11e935), [BNB](https://app.safe.global/home?safe=bnb:0xC18F11735C6a1941431cCC5BcF13AF0a052A5022), [Optimism](https://app.safe.global/home?safe=oeth:0x75483CE83100890c6bf1718c26052cE44e0F2839), [Polygon](https://app.safe.global/home?safe=matic:0xC18F11735C6a1941431cCC5BcF13AF0a052A5022), [Scroll](https://app.safe.global/home?safe=scr:0xC18F11735C6a1941431cCC5BcF13AF0a052A5022))
-`0xADB90Cfb3d5ebbaB8eeE7DA10B4DB215A7d50BeE` - [zksync](https://app.safe.global/home?safe=zksync:0xADB90Cfb3d5ebbaB8eeE7DA10B4DB215A7d50BeE)
+`0xADB90Cfb3d5ebbaB8eeE7DA10B4DB215A7d50BeE` - [zksync](https://app.safe.global/home?safe=zksync:0xADB90Cfb3d5ebbaB8eeE7DA10B4DB215A7d50BeE
```
Comment (AlexLido) [inline docs/multisigs/committees.md]: Missing closing parenthesis

**Review by AlexLido — CHANGES_REQUESTED** [review]
Please double check my comments before merging, otherwise lgtm

File: docs/multisigs/committees.md (hunk 1 above)
Comment (kate-aleksseeva) [reply docs/multisigs/committees.md]: fixed

File: docs/multisigs/committees.md (hunk 2 above)
Comment (kate-aleksseeva) [reply docs/multisigs/committees.md:127]: It was decided to remove line breaks in cases where a single link is specified for a forum post, a Snapshot, and an Aragon.

File: docs/multisigs/committees.md (hunk 3 above)
Comment (kate-aleksseeva) [reply docs/multisigs/committees.md]: It was decided to remove line breaks in cases where a single link is specified for a forum post, a Snapshot, and an Aragon.

File: docs/multisigs/committees.md (hunk 4 above)
Comment (kate-aleksseeva) [reply docs/multisigs/committees.md:129]: It was decided to remove line breaks in cases where a single link is specified for a forum post, a Snapshot, and an Aragon.

**Review by AlexLido — APPROVED** [review]
lgtm

**Pull request merged**